Common/Tests/Utilities/MSTestEnvironment.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Common = Microsoft.PythonTools.Infrastructure;
+using Analysis = Microsoft.PythonTools.Analysis.Infrastructure;
```

Comment:
This should go in `TestUtilities.Python.Analysis` in order to take dependencies on PythonTools assemblies. (Or we should move the entire TestUtilities project out of the Common directory - I'm 99% sure nobody else is using this project now, but at one point a couple of external companies were using it.)

Reply:
It is dependent only by projects in `Test` folder:

![image](https://user-images.githubusercontent.com/8933858/36053205-a0775a00-0da5-11e8-8c2d-359c55b547ec.png)

Do you want me to move it?

Reply:
Within our repo, this is true, but there are other projects besides PTVS that use this. If we move it, they have to stop using it, whereas if we just move the new additions into the Python-specific project then it doesn't affect the Common one.

Reply:
Ok. So `TestUtilities.Python.Analysis` or `TestUtilities.Python`?

Reply:
TestUtilities.Python.Analysis (which is the "non-UI" test project... the naming is not great :) )